kickstarter/android-oss
# [MBL-1060] Add dark mode toggle in settings, make settings visible to non logged in users

# 📲 What

Add a shared prefs value for the selected app theme mode (match system[0], light[1], or dark[2])

# 🤔 Why

Users should have a way to set the app theme once we are ready to let them use dark mode (defaults to system)

# 🛠 How

Add a shared prefs value (default pulls to match system)

# 👀 See

![Screenshot_20231114-163732](https://github.com/kickstarter/android-oss/assets/7563288/75717d93-e1f2-4de7-9bbf-d7a1fdbfb4b6)
![Screenshot_20231114-163737](https://github.com/kickstarter/android-oss/assets/7563288/5d9b8aab-c114-466a-a431-6ae6aff73dd1)
![Screenshot_20231114-163728](https://github.com/kickstarter/android-oss/assets/7563288/fb76a02c-4af4-4d2f-9151-279aef7e0ada)


# 📋 QA

1) Log in
2) Go to settings
3) Change mode to desired choice
4) GO to search or login flow to check selection changed theme
5) Log out
6) Do it again

# Story 📖

[MBL-1060](https://kickstarter.atlassian.net/browse/MBL-1060)


[MBL-1060]: https://kickstarter.atlassian.net/browse/MBL-1060?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
